learn/develop/models/index.qmd

```diff
@@ -516,26 +516,25 @@ show_best(mda_tune_res, metric = "roc_auc")
 
 There are various things that came to mind while developing this resource.
 
-**Do I have to return a simple vector for `predict` and `predict_class`?**
+**Do I have to return a simple vector for `predict()`?**
 
-Previously, when discussing the `pred` information:
+There are some models (e.g. `glmnet`, `plsr`, `Cubist`, etc.) that can make predictions for different models from the same fitted model object. We facilitate this via `multi_predict()`, rather than `predict()`.
```

Comment:
Are `glmnet`, `plsr`, `Cubist` referring to packages or engines? if engines they should be in quotes

Reply:
These are not the engine names. I'm gonna remove the code styling instead (assuming that the general names are better known than the engine names.)